src/components/event_view/event_info/EventInfo.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'react-bootstrap';
+import EventField from '../event_fields/EventField';
+import EventFieldEdit from '../event_fields/EventFieldEdit';
+
+export class EventInfo extends Component {
```

Comment:
Just to clarify, this doesn't replace the AccomInfo, TranInfo, and PlanInfo components—it just supplements them by putting their common elements in one component? A comment here would help. I really like how this code only appears once! I'm not 100% sold on all the separate AccomInfo, TranInfo, and PlanInfo components in addition, but I can't think of a better way to do it, so I have a hunch this may be the best way to go!

Reply:
Correct.